src/parsers/manifestjson.js

```diff
@@ -306,13 +335,28 @@ export default class ManifestJSONParser extends JSONParser {
       match &&
       baseObject.code !== messages.MANIFEST_BAD_PERMISSION.code &&
       baseObject.code !== messages.MANIFEST_BAD_OPTIONAL_PERMISSION.code &&
+      baseObject.code !== messages.MANIFEST_BAD_HOST_PERMISSION.code &&
       baseObject.code !== messages.MANIFEST_PERMISSION_UNSUPPORTED
     ) {
       baseObject = messages[`MANIFEST_${match[1].toUpperCase()}`];
       overrides.message = oneLine`/${match[1]}: Unknown ${match[1]}
           "${error.data}" at ${match[2]}.`;
     }
 
+    // Make sure we filter out warnings and errors code that should never be reported
+    // on manifest version 2 extensions.
+    const ignored_on_manifestV2 = [
```

Comment:
```suggestion
    const ignoredOnMV2 = [
```

Reply:
(it's mostly about using camelcase)

Reply:
:+1: I'm not sure why I didn't make it in camelcase :shrug: